core/services/gateway/handlers/capabilities/handler.go

```diff
@@ -177,12 +179,81 @@ func (h *handler) handleWebAPITriggerMessage(ctx context.Context, msg *api.Messa
 	return nil
 }
 
+func (h *handler) handleComputeActionMessage(ctx context.Context, msg *api.Message, nodeAddr string) error {
```

Comment:
Is there any difference between the compute gateway handler + and target one?

Also I can see you're using sdk.FetchResponse here: I think you be using capabilities.Response here, and translate between the two in the Fetcher

i.e. the connector handler + gateway handler should know nothing about sdk.FetchRequest/sdk.FetchResponse

Reply:
you are right, I modified this on the outgoing_connector_handler, but not on the gateway side. Changed now 👍🏼